source/getting_started.rst

```diff
@@ -3,8 +3,15 @@
 Getting Started
 ===============
 
+.. figure:: _static/coming-soon.png
+   :align: center
+
+   **Coming Soon**
+
+.. sectionauthor:: Manav
```

Comment:
This directive should be the first line in the file, followed by any placeholder and owner comments and suggestions

Reply:
Changing this in all files will be not good use of time, shall we ask the respective authors to make the changes as they will remove the coming soon anyways?

Reply:
Why will it not be a good use of time? It will take less than a minute per file. Let's do it right now instead of relying on authors to do it, which they may or may not do, and I may not remember to check it then.

Reply:
Done.